### ISSUE TITLE
Search input clear button cannot receive keyboard focus (WCAG Level A)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2237]
The 'x' added by the browser to input type `search` to clear the input, cannot be focussed by a keyboard user.

(Page 47 of June 2022 DS DAC report)...

---

### Issue ID: DAC_Keyboard_Access_02
URL: https://ons-design-system-prototypes.netlify.app/prototypes/dac-may2022/checkboxes-3.html
Page title: Search, simple accordion, revealed checkboxes and document list
Journey: Journey 5, Step 14

### Screenshot:
![Screenshot 2022-06-16 at 09 27 41](https://user-images.githubusercontent.com/43346934/174028683-de823f0b-83d6-43e5-af00-fc2f59ea69f3.png)

A pseudo ‘X’ (clear) element appears within the ‘Search’ input while it has mouseover or keyboard-only focus. Keyboard-only users are unable to navigate to the element that can be activated to clear the text within the search input.

### Solution:
Elements that can be activated with mouse inputs must also be able to gain keyboard-only focus and be activated by keyboard-only inputs. We recommend that an additional button is added next to the search input that can be navigated onto by keyboard-only users and activated to clear the text within the search input.
